src/formatDate.js

```diff
@@ -8,7 +8,56 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
```

Comment:
Be cautious when using `Array.prototype.pop` as it mutates the original array. If `fromFormat` or `toFormat` are needed later in their original form, this could lead to bugs. Consider using a different method to access the last element without modifying the array.